test/providers/keychain/start

```diff
@@ -1,33 +1,3 @@
 #!/bin/bash
 
-set -euo pipefail
-
-# clear out previously created test variables
-echo "Deleting prior password values stored for this test..."
 ./stop
```

Comment:
Should we add a comment saying "Clear out previously store secrets from keychain"
(as a quick explanation why a `start` script is only doing a `stop`)?

Reply:
I'll add it.